pandas/tests/generic/test_finalize.py

```diff
@@ -678,6 +678,8 @@ def test_datetime_method(method):
         "microsecond",
         "nanosecond",
         "dayofweek",
+        # Add better named field for GH-9606
```

Comment:
nit-pick but we usually don't leave comments like this: git/blame keeps track of when changes occurred